Source/WebKit/GPUProcess/mac/GPUProcessMac.mm

```diff
@@ -67,7 +67,9 @@
 void GPUProcess::updateProcessName()
 {
 #if !PLATFORM(MACCATALYST)
-    RetainPtr applicationName = adoptNS([[NSString alloc] initWithFormat:WEB_UI_NSSTRING(@"%@ Graphics and Media", "visible name of the GPU process. The argument is the application name."), m_uiProcessName.createNSString().get()]);
+ALLOW_NONLITERAL_FORMAT_BEGIN
```

Comment:
What is this about?

Reply:
This is done to fix the [alpha.webkit.UnretainedCallArgsChecker] warning on this line, I am not too sure though about the NONLITERAL stuff. What do you think we should do for WEB_UI_NSSTRING, which is used a lot, and defers to WebCore::localizedNSString, which seems not a trivial function?

Reply:
@rniwa can you check this part? I had it removed from the patch you reviewed, let me know if there is a better way to fix this (likely there will be more spots like this).